hm-gtm.php

```diff
@@ -18,33 +18,53 @@
  * Output the gtm tag, place this immediately after the opening <body> tag
  *
  * @param bool $echo
+ * @param bool $network True to include network wide Google Tag Manager
+ *
  * @return string
  */
-function tag( $echo = true ) {
+function tag( $echo = true, $network = true ) {
```

Comment:
I'd leave out the network flag myself, not sure what use cases there are where you wouldn't want it if available.